core/shared/src/main/scala/cats/parse/BitSet.scala

```diff
@@ -22,12 +22,12 @@
 package cats.parse
 
 import java.util.BitSet
-
 object BitSetUtil {
   type Tpe = BitSet
-
+  @deprecated("", "0.3.7")
```

Comment:
I changed  here to `isScalaJVM =false , isScalaJs = false` because I think combination of `isScalaJVM =true, isScalaJs = false`, `isScalaJVM =false, isScalaJs = true` or `isScalaJVM =true, isScalaJs = true` does not make sense.

Reply:
> instead we could move the BitSet code to ...

Yeah, actually, I thought my idea was not nice too😓

That seems better solution! Thanks!